### ISSUE TITLE
Running a custom generator?

### ISSUE THREAD

**Issue by fullstackwebdev** [deebloo/yeoman-blueprints#1]
Hello, thanks again for this project.

How do I run a generator, after I created it in the "./generators" directory?

I created one called "super" with some blank files (blueprint.json having {})

```
 tree ./generators 
./generators
└── super
    └── templates
        ├── blueprint.json
        └── controller
            └── template.js

3 directories, 2 files
```

However, I don't exactly see how to run it ... are these suppose to be in their own yeoman generator that I need to install? confused, thanks

**Comment by deebloo** [comment]
yes. this repo is the guts of how the overridable templates work in generator angular blueprint. you you will have to create your own yeoman generator and then you can use blueprints.NamedBase instead of yeoman.NamedBase
